docs/features/aliases.md

```diff
@@ -1 +1,17 @@
-# Aliases
\ No newline at end of file
+# Aliases
+
+```js
+// This will create a simple alias for a record type
+alias test = {
+	a:int,
+	b:int,
+	c:str,
+}
+
+let test2:test = {a:1, b:1, c:"test"}
+// This is useful for shortening down big types when you want to make it clear what type it is based off of code
+```
+
+## Notes:
+- You are unable to add the `pub` modifier to `alias` because it is a type and cannot be exported.
```

Comment:
I believe you can export aliases

Reply:
Oh, ok, I just checked, idk where I got that from.